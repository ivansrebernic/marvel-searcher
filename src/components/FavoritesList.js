import React from 'react'
import styled from 'styled-components'


const FavoritesListStyled = styled.ul`
    position:absolute;
    background-color:white;
    width:80vw;
    top:100%;
    margin:0;
    padding:0;
    border-radius: 0 0 5px 5px;
    @media (max-width: 768px) {
        left:0;
        width:100%;
      }
`


function FavoritesList(props) {



    return (

        <FavoritesListStyled  >
            {props.children}
        </FavoritesListStyled>

    )


}

export default FavoritesList