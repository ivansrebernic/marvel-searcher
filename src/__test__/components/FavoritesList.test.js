import React from 'react'
import { shallow } from 'enzyme'
import FavoritesList from '../../components/FavoritesList'


describe('<FavoritesList>', () => {
    it('Renders correctly', () => {
        const wrapper = shallow(<FavoritesList></FavoritesList>)
        expect(wrapper).toMatchSnapshot()
    })

})