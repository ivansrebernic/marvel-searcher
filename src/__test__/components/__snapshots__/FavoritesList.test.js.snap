// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FavoritesList> Renders correctly 1`] = `<styled.ul />`;
